egs/tedlium/s5_r3/local/chain/tuning/run_tdnnf_1a.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+
+# run_tdnnf_1a.sh is the script which results are presented in the corpus release paper.
+# It use 2 to 6 jobs and add proportional-shrink 10.
+
+# local/chain/compare_wer_general.sh exp/chain_cleaned/tdnn_1a exp/chain_cleaned/tdnnf_1a
+# System                    tdnn_1a   tdnnf_1a
+# WER on dev(orig)            8.2       7.9
+# WER on dev(rescored)        7.6       7.2
+# WER on test(orig)           8.1       8.0
+# WER on test(rescored)       7.7       7.5
+# Final train prob          -0.0802   -0.0779
+# Final valid prob          -0.0980   -0.0906
+# Final train prob (xent)   -1.1450   -0.9021
+# Final valid prob (xent)   -1.2498   -0.9971
+
+
+## how you run this (note: this assumes that the run_tdnn.sh soft link points here;
+## otherwise call it directly in its location).
+# by default, with cleanup:
+# local/chain/run_tdnn.sh
+
+# without cleanup:
+# local/chain/run_tdnn.sh  --train-set train --gmm tri3 --nnet3-affix "" &
+
+# note, if you have already run the corresponding non-chain nnet3 system
+# (local/nnet3/run_tdnn.sh), you may want to run with --stage 14.
+
+
+set -e -o pipefail
+
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+nj=30
+decode_nj=30
+min_seg_len=1.55
+xent_regularize=0.1
+train_set=train_cleaned
+gmm=tri3_cleaned  # the gmm for the target data
+num_threads_ubm=32
+nnet3_affix=_cleaned  # cleanup affix for nnet3 and chain dirs, e.g. _cleaned
+
+# The rest are configs specific to this script.  Most of the parameters
+# are just hardcoded at this level, in the commands below.
+train_stage=-10
+tree_affix=  # affix for tree directory, e.g. "a" or "b", in case we change the configuration.
+tdnn_affix=f_1a  #affix for TDNN directory, e.g. "a" or "b", in case we change the configuration.
```

Comment:
i'd rather you make this 1a and call it tdnnf_affix or just model_affix and change how you use it.